process/lifecycle.md

```diff
@@ -9,69 +8,205 @@ This lifecycle document is maintained by The Academy Software Foundation (“ASW
+## Sandbox Stage
+
+Projects being submitted to the ASWF at the sandbox level are intended to be the entry point for early-stage projects. Characteristics for projects at the Sandbox Stage maybe one or more of:
+
+* Early-stage projects that the ASWF TAC believes warrant experimentation.
+* New projects that are designed to extend one or more TAC projects with functionality or interoperability libraries.
+* Independent projects that fit the ASWF mission/vision and provide the potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need).
+* Projects commissioned or sanctioned by the ASWF, including initial code for ASWF Working Group collaborations, and "experimental" projects.
+* Any project that realistically intends to join ASWF Incubation in the future and wishes to lay the foundations for that.
+
+### Requirements
 
 To be accepted at the Incubation stage, a project must:
 
 * Submit a completed [Project Contribution Proposal Template](proposal_template.md) to the TAC, or the TAC’s designated recipient for contribution proposals.
 * Provide such additional information as the TAC may reasonably request.
 * Be available to present to the TAC with respect to the project’s proposal and inclusion in ASWF. Project teams should be prepared to present a detailed (20-30 minutes in length) overview on the project in addition to speaking to the information contained in the project contribution proposal.
 * Be deemed by the TAC to add potential value or value to the mission of ASWF.
-* Have a technical charter that provides for inbound and outbound licensing of code under an OSI-approved license approved by the Governing Board of ASWF for projects. The ASWF maintains a [template](charter_template.md) for projects to use.
-* Agree to transfer any relevant trademarks to an LF entity to hold for the project. In the case of projects with established trademarks where a trademark transfer is commercially difficult, we generally recommend the project use a new name upon incubation.
 * Obtain an affirmative vote of the TAC.
 
-An initial review of a proposal submitted to the TAC should be conducted within two to four weeks following acceptance of a project contribution proposal. If a project is a new project, it’s often helpful to share the vision for the project, the anticipated structure and benefits.
-If the project is an existing open source project proposed to join ASWF, it is often helpful to schedule a project walkthrough for the community to better understand the project, including the architecture, structure and how to get starting using the project. This often helps progress reviews more efficiently.
+### Benefits
 
-## Graduation Review
+The Sandbox Stage benefits are outlined below.
 
-To be advance to an Adopted stage, a project must meet the Incubation stage requirements plus:
+* Neutral hosting of the project’s community and any key assets (e.g. trademark, domain, etc.).
```

Comment:
Previous language about Incubation Stage requirements had a line about transferring trademarks, domain... which got removed for Sandbox Stage, but this talks about "neutral hosting of trademarks and domains". Is this indeed the right stage to do this?

Reply:
Yes - any new assets will be neutrally hosted anywhere in this lifecycle. Existing ones should transition as the project reaches the Incubation stage. Let me know if this answers your question

Reply:
Seems like a lot of the things that need a sandbox are so preliminary that they probably have not accumulated any property. If the project exists enough to have web sites, trademarks, or even a pile of code, it's probably ready for incubation.